src/Splat/Platforms/Android/Bitmaps/PlatformBitmapLoader.cs

```diff
@@ -69,6 +74,11 @@ public async Task<IBitmap> Load(Stream sourceStream, float? desiredWidth, float?
         /// <inheritdoc />
         public Task<IBitmap> LoadFromResource(string source, float? desiredWidth, float? desiredHeight)
         {
+            if (_drawableList == null)
```

Comment:
What is the current behavior here if this case wasn't caught?  Is this potentially a breaking change?

Reply:
Not a breaking change. The current behavior is that the user doesn’t even get a chance to call the method. Because it crashes in the constructor when accessing BitmapLoader.Current.